src/Operation/RemoteMergeOperation.php

```diff
@@ -143,6 +147,21 @@ public function pushToRemote()
         $this->gitHelper->pushToRemote($this->targetRemote, $target);
     }
 
+    public function rebase(bool $rebase = false)
+    {
+        $this->rebase = $rebase;
+        $this->guardSync = !$rebase;
```

Comment:
Maybe the command should state that `--rebase` explicitly disables `--ensure-sync`, or a check should be added to ensure both aren't used together